draft-ietf-wimse-s2s-protocol.md

```diff
@@ -228,38 +228,21 @@ defines the "wimse" URI scheme which can be used by any deployment that implemen
 An example WIT might look like this:
 
 ~~~ jwt
-{::include includes/wit.txt.out}
+{::include includes/thin-wit.txt.out}
 ~~~
 {: #example-wit title="An example Workload Identity Token (WIT)"}
 
 The decoded JOSE header of the WIT from the example above is shown here:
 
 ~~~ json
-{
-  "alg": "ES256",
-  "kid": "June 5",
-  "typ": "wimse-id+jwt"
-}
+{::include includes/thin-wit-header.txt.out}
 ~~~
 {: title="Example WIT JOSE Header"}
 
 The decoded JWT claims of the WIT from the example above are shown here:
 
 ~~~ json
-{
-  "cnf": {
-    "jwk": {
-      "alg": "EdDSA",
-      "crv": "Ed25519",
-      "kty": "OKP",
-      "x": "1CXXvflN_LVVsIsYXsUvB03JmlGWeCHqQVuouCF92bg"
-    }
-  },
-  "exp": 1745512510,
-  "iat": 1745508910,
-  "jti": "bd2a7b5bf8573a41adb4cbf3cfa01e15",
-  "sub": "wimse://example.com/specific-workload"
-}
+{::include includes/thin-wit-claims.txt.out}
```

Comment:
pretty sure this introduces or reintroduces inconsistencies between example content and text down a few lines but probably/maybe other places too